test/verify/check-system-info

```diff
@@ -193,6 +193,10 @@ class TestSystemInfo(MachineCase):
         b.wait_in_text('#system_information_hostname_text', "Adventure Box (host1.cockpit.lan)")
         self.assertEqual(m.execute("hostname").strip(), "host1.cockpit.lan")
 
+        m.execute("hostnamectl set-hostname --static ''")
+        m.execute("hostnamectl set-hostname 'mydhcpname'")
```

Comment:
Oh wait, that's cheating -- `set-hostname` also writes /etc/hostname. Please do `--transient`.

Reply:
Yup.. ha! Setting --transient after only static to empty does not work either, as the `Pretty Hostname` is still set. So I've first emptied everything and then set the transient hostname in the test.